packages/p2p/stores/general-store.js

```diff
@@ -1,8 +1,9 @@
 import { observable, action, runInAction } from 'mobx';
-import { isEmptyObject, epochToMoment, getSocketURL } from '@deriv/shared';
-import { orderToggleIndex } from 'Components/orders/order-info.js';
+import { epochToMoment, getSocketURL, isEmptyObject, mobileOSDetect, routes } from '@deriv/shared';
+import { localize, Localize } from 'Components/i18next';
 import { createExtendedOrderDetails } from 'Utils/orders.js';
 import { init as WebsocketInit, requestWS, subscribeWS } from 'Utils/websocket.js';
+import { order_list } from '../src/constants/order-list';
```

Comment:
Can use alias here `Constants/order-list`, you can refer to `webpack.config.js`

Reply:
Currently stores are outside `src` folder, we can do this when we move them back.